sql/analyzer/analyzer.go

```diff
@@ -142,8 +143,9 @@ func (a *Analyzer) Log(msg string, args ...interface{}) {
 
 // Analyze the node and all its children.
 func (a *Analyzer) Analyze(ctx *sql.Context, n sql.Node) (sql.Node, error) {
-	span, ctx := ctx.Span("analyze")
-	span.LogKV("plan", n.String())
+	span, ctx := ctx.Span("analyze", opentracing.Tags{
+		"plan": n.String(),
```

Comment:
What is the difference between `LogKV` and `Tags`?

Reply:
Tags identify the span. Logs are things that happen on span execution.